assets/src/edit-story/components/form/dateTime/index.js

```diff
@@ -24,10 +37,19 @@ const DateTimeWrapper = styled.div`
 `;
 
 function DateTime({ value, onChange, is12Hour = true, forwardedRef }) {
+  const selectedTime = value ? new Date(value) : new Date();
+  const initialHours = selectedTime.getHours();
+  const [localData, setLocalData] = useState({
```

Comment:
I still don't quite understand value vs state behavior here. What happens if the `value` prop changes? Will it stick to the old `localData`? I know we are a bit pressed with timing on the replacement. And, IMHO, if we can't find any other easy way, we could even just do `useEffect(() => setLocalState(prop), [prop])` kind of thing. Not sure if @barklund has any advice there.

Reply:
Currently, it would use`localData`. Not sure if it shouldn't though -- if the user is editing the value in the calendar then perhaps that data should be the one used (as now), and not the value that changes from somewhere else for some reason. Thoughts?

Reply:
I don't see the value changing outside this component as a big issue here. Seems rather unlikely.